apps/extension/src/Setup/ImportAccount/SeedPhraseImport.tsx

```diff
@@ -47,27 +47,31 @@ export const SeedPhraseImport: React.FC<Props> = ({ onConfirm }) => {
     Array.from(mnemonicsRange)
   );
 
-  const privateKeyError = (() => {
-    const validation = validatePrivateKey(filterPrivateKeyPrefix(privateKey));
+  const validatePkAndFormatErrorMessage = (key: string): string => {
```

Comment:
I think it's fine to call it `validatePk`. We can also return `{valid: boolean, msg: string}` or something, as it seems a bit more explicit :)

Reply:
sure. updated function name and return type 👍